arrow/util.py

```diff
@@ -23,6 +28,42 @@ def is_timestamp(value):
         return False
 
 
+def datetime_from_timestamp(timestamp, tz=None):
+    """Computes datetime from timestamp. Supports negative timestamps on Windows platform."""
+    sec_frac, sec = math.modf(timestamp)
+    dt = datetime.datetime(1970, 1, 1, tzinfo=dateutil.tz.tzutc()) + datetime.timedelta(
+        seconds=sec, microseconds=sec_frac * 1e6
+    )
+    if tz is None:
```

Comment:
Hmmm I think arrow will generally set timezones to UTC rather than local if no info is provided, @jadchaar any thoughts?

Reply:
Yeah I think you are correct. This should probably be UTC by default rather than local.

Reply:
Ok, cool